### ISSUE TITLE
Switch View button not focusable + Fix order of buttons that are focused upon clicking tab on Hearings schedule pages

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12125]
After the Apply button the Switch view element does not receive focus. Focus skips to the Download Current View element. (New: 08/27/2019) The focus does shift around in the wrong order on this page in particular

- Switch View button is not being read by AT
- The order of which buttons are focused on while navigating the page via tab is strange and should be made more orderly (i.e. focus on ""Schedule Veterans"" before ""Build Schedule"" because it is visually to the left)"
